docs/machineset-aws.md

```diff
@@ -4,7 +4,7 @@ _\<windows_container_ami\>_ should be replaced with the AMI ID of a Windows imag
  You must use Windows Server 2019 with a version 10.0.17763.1457 or earlier. Run the following command to list AWS image info:
   ```shell script
   $ aws ec2 describe-images \
-    --filters Name=name,Values=Windows_Server-2019-English-Full-ContainersLatest-2020.09.09 \
+    --filters Name=name,Values=Windows_Server-2019-English-Full-ContainersLatest-2021.01.13 \
```

Comment:
This means that we will be moving away from `10.0.17763.1457`. But it looks like we cannot pin to that kernel version anymore :-( @openshift/openshift-team-windows-containers how should we proceed?

@codyhoag are we mentioning this command anywhere in the OpenShift docs? Or was it just in the release notes?

Reply:
Should we test newer kernel versions for the issue and see if it still persists?

Reply:
Yes we should. But it should be of lower priority as of now.

Reply:
@aravindhp this is only mentioned in the release notes as part of the [known issue](https://docs.openshift.com/container-platform/4.6/windows_containers/windows-containers-release-notes.html#wmco-1-0-1-known-issues).

Reply:
@codyhoag I assume we don't need to retroactively keep updating that. So we should be fine from a doc perspective.

Reply:
@alinaryan please remove `Run the following command to list AWS image info:` and the shell script section.